ngx_http_pipelog_module.c

```diff
@@ -2247,7 +2247,7 @@ ngx_http_pipelog_command_exec (ngx_str_t *command, ngx_fd_t rfd, ngx_cycle_t *cy
     ngx_pid_t pid;
     char cmd[1024];
     ngx_fd_t fd;
-    wordexp_t  w;
+    ngx_http_pipelog_pim_t pim;
```

Comment:
It seems to be an uninitialized variable. I would suggest changing the first argument of `ngx_http_pipelog_command_exec()` to `wordexp_t *` and passing `&pim[idx].w` on the caller side.

Reply:
It's going to be a few weeks before I can get to this due to the holidays.